ygen/gogen_test.go

```diff
@@ -1366,11 +652,174 @@ func (*Tstruct) ΛBelongingModule() string {
 }
 `,
 		},
-		wantUncompressed: wantGoStructOut{
```

Comment:
Did we decide that we should not have testing for uncompressed schemas here? I don't immediately see any tests that replaced this.

Reply:
There is no such thing as compressed vs. uncompressed once we have the IR as far as `writeGoStruct` is concerned, that is handled all at IR generation time.

Reply:
Makes sense :-D, thanks, apologies I missed this.